src/core/datatypes/phylogenetics/TreeDiscreteCharacterData.h

```diff
@@ -23,7 +23,7 @@ namespace RevBayesCore {
         
     public:
         TreeDiscreteCharacterData();                                                                                                    //!< Default constructor
-        
+//        ~TreeDiscreteCharacterData();                                                                                                    //!< Default constructor
```

Comment:
Maybe delete the comment if you don't want the destructor?

Reply:
Done.